lua/terrortown/lang/zh_tw.lua

```diff
@@ -2090,5 +2090,11 @@ L.search_eyes = "透過你的探查技能，你確信他臨死前見到的最後
 --L.entity_pickup_owner_only = "Only the owner can pick this up"
 
 -- 2023-12-18
-L.body_confirm = "{finder} 確認了 {victim} 的死。"
+--L.body_confirm_one = "{finder} confirmed the death of {victim}."
```

Comment:
This should stay and only be renamed

Reply:
i had a fear that this would happen in the middle of one of them, corrected now